less/navigation.less

```diff
@@ -1 +1,47 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+.navigation {
+    margin: 0 auto;
+    background-color: @wild-sand;
```

Comment:
We talked about this already. But in order to center a fixed item in the middle of the page. It's top:50%; left: 50%; then use transform: transition(-50%, -50%);